docs/next/api/introduction.md

```diff
@@ -11,7 +11,7 @@ canonicalUrl: /api/
 
 Welcome to Handsontable API reference. Our goal is make it easy to dive in and start coding from day one.
 
-However, data grids are rather complex libraries, so we assume that you possess a certain level of expertise in JavaScript before moving forward. If you want to grab some basics, we have provided a JavaScript example in the [Hello World](../hello-world/) section.
+However, data grids are rather complex libraries, so we assume that you possess a certain level of expertise in JavaScript before moving forward. If you want to grab some basics, we have provided a JavaScript example in the [Hello World](@/hello-world/) section.
```

Comment:
I suppose that is incorrect. Could you check this link once again?

Reply:
Afaik: the proper link is `[Hello World](@/guides/hello-world.md)`

Reply:
Thank you. The correct link that I tested is in e23c4afd59a15eebee5c66ae443bed06fdd51a6a.